app/src/main/java/com/realityexpander/tasky/core/presentation/notifications/RemindAtAlarmNotificationManager.kt

```diff
@@ -0,0 +1,287 @@
+package com.realityexpander.tasky.core.presentation.notifications
+
+import android.Manifest
+import android.app.AlarmManager
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.text.Layout
+import android.text.StaticLayout
+import android.text.TextPaint
+import androidx.core.app.ActivityCompat
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.realityexpander.tasky.BuildConfig
+import com.realityexpander.tasky.MainActivity
+import com.realityexpander.tasky.R
+import com.realityexpander.tasky.agenda_feature.presentation.common.enums.AgendaItemType
+import com.realityexpander.tasky.agenda_feature.presentation.common.enums.toAgendaItemType
+import com.realityexpander.tasky.core.presentation.util.getBitmapFromVectorDrawable
+import com.realityexpander.tasky.core.util.UuidStr
+import com.realityexpander.tasky.core.util.toIntegerHashCodeOfUUIDString
+import com.realityexpander.tasky.core.util.toUtcMillis
+import com.realityexpander.tasky.core.util.toZonedDateTime
+import logcat.logcat
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.util.*
+
+
+
+interface IRemindAtAlarmNotificationManager {
```

Comment:
Usually the interfaces are declared on the domain package, and have their implementations on the data layer, since usually the interface is used by something on the domain layer

Reply:
OK, I have moved them to the `domain` for `agenda_feature`